docs/_build/html/_sources/tutorial.md.txt

```diff
@@ -0,0 +1,71 @@
+# Tutorial: First 15 Minutes
+This tutorial walks you through setting up Tilt for your project. It should take 15 minutes, and assumes you've already [installed Tilt](install.html). Before you begin, you may want to join the `#tilt` channel in [Kubernetes Slack](http://slack.k8s.io) for technical and moral support.
```

Comment:
worth noting that we assume they have k8s yaml/dockerfiles? if there's a trend of ppl getting over-excited about k8s cuz it's the new hotness, i can imagine someone coming to this tutorial thinking that tilt will help them containerize/k8s-ify their app.